.eslintrc

```diff
@@ -72,26 +72,20 @@
     "no-nested-ternary": "error",
     "no-unneeded-ternary": "error",
     "camelcase": "off",
-    "arrow-parens": ["off", "as-needed"],
     "no-debugger": "error",
     "no-empty": "error",
-    "no-underscore-dangle": "error",
     "no-unused-labels": "error",
     "prefer-const": "error",
-    "no-unused-vars": "off",
-    "jsx-quotes": ["error", "prefer-double"],
-    "@typescript-eslint/no-unused-vars": [
+    "@typescript-eslint/no-unused-vars": "off",
+    "unused-imports/no-unused-imports": "error",
+    "unused-imports/no-unused-vars": [
       "warn",
-      {
-        "ignoreRestSiblings": true
-      }
+      { "vars": "all", "varsIgnorePattern": "^_", "args": "after-used", "argsIgnorePattern": "^_" }
     ],
-    "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-non-null-assertion": "warn",
     "@typescript-eslint/no-explicit-any": "warn",
     "@typescript-eslint/no-duplicate-enum-values": "warn",
-    "padding-line-between-statements": [
```

Comment:
To enhance code readability, it's commonly suggested to include a blank line between your code and a return statement. Implementing this practice can contribute to better code understanding.

Reply:
I removed this rule because ESLint deprecated and removed it to a separate plugin in the next version. The reason they removed it is because it has to do with opinionated code formatting, for which we are using prettier.

My personal preference is for keeping this so I'll revert for now and see what others think